themes/agnoster/agnoster.base.bash

```diff
@@ -292,7 +292,7 @@ __command_rprompt() {
         [ $n -gt 40 ] || break
         times="$times ${tz%%:*}\e[30;1m:\e[0;36;1m"
         times="$times$(TZ=${tz#*:} date +%H:%M)\e[0m"
-        n=$(( $n - 10 ))
+        n="$( $n - 10 )"
```

Comment:
No way. This is an arithmetic expansion but not trying to execute a command with the name $n. Also, quoting is unneeded.

Reply:
Rejected.